<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>295</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:app.ico</normaloff>:app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="horizontalSpacing">
     <number>1</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="pboman3::ui::TreeWidget" name="treeWidget">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="actionFileClose"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
    <addaction name="actionCheckUpdates"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelectionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionExtractTo"/>
    <addaction name="actionSelectionExtractFolder"/>
    <addaction name="actionSelectionExtractContainer"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionRename"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionCut"/>
    <addaction name="actionSelectionCopy"/>
    <addaction name="actionSelectionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectionDelete"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewHeaders"/>
    <addaction name="actionViewSignature"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="pboman3::ui::StatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>Open..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSelectionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </action>
  <action name="actionSelectionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSelectionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelectionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectionRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSelectionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionViewHeaders">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Headers</string>
   </property>
  </action>
  <action name="actionViewSignature">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Signatrue</string>
   </property>
  </action>
  <action name="actionSelectionExtractTo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Extract to...</string>
   </property>
  </action>
  <action name="actionSelectionExtractFolder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Extract to...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSelectionExtractContainer">
   <property name="text">
    <string>Extract to...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCheckUpdates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pboman3::ui::TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>treewidget/treewidget.h</header>
  </customwidget>
  <customwidget>
   <class>pboman3::ui::StatusBar</class>
   <extends>QStatusBar</extends>
   <header>statusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
